client/src/components/search/searchPage/SearchPageHits.js

```diff
@@ -9,65 +10,77 @@ import { isEmpty } from 'lodash';
 
 import EmptySearch from './EmptySearch';
 import NoResults from './NoResults';
-import { homeLocation } from '../../../../config/env.json';
 
 import './search-page-hits.css';
 
-const indexMap = {
-  challenges: {
-    title: 'Lesson',
-    url: `${homeLocation}/learn`
-  },
-  guides: {
-    title: 'Guide',
-    url: `${homeLocation}/guide`
-  },
-  youtube: {
-    title: 'YouTube',
-    url: 'https://www.youtube.com/watch?v='
-  }
-};
-
-const buildUrl = (index, result) =>
-  `${indexMap[index].url}${'videoId' in result ? result.videoId : result.url}`;
-
 const AllHits = connectAutoComplete(({ hits, currentRefinement }) => {
-  if (hits.some(hit => isEmpty(hit.index))) {
-    return null;
-  }
-  const nonQuerySuggestionHits = hits.filter(
-    ({ index }) => index !== 'query_suggestions'
-  );
-  const isHitsEmpty = nonQuerySuggestionHits.every(({ hits }) => !hits.length);
+  const isHitsEmpty = !hits.length;
 
   return currentRefinement && !isHitsEmpty ? (
     <div className='ais-Hits search-page'>
       <ul className='ais-Hits-list'>
-        {nonQuerySuggestionHits.map(({ hits: results, index }) =>
-          results.map(result => (
-            <a
-              href={buildUrl(index, result)}
-              key={result.objectID}
-              rel='noopener noreferrer'
-              target='_blank'
-            >
-              <li className='ais-Hits-item dataset-node'>
-                <p>
-                  <strong>{indexMap[index].title}:</strong>
-                  &nbsp;
-                  <Highlight attribute='title' hit={result} />
-                </p>
-              </li>
-            </a>
-          ))
-        )}
+        {hits.map(result => (
+          <a
+            href={result.url}
+            key={result.objectID}
+            rel='noopener noreferrer'
+            target='_blank'
+          >
+            <li className='ais-Hits-item dataset-node'>
+              <p>
+                <Highlight attribute='title' hit={result} />
+              </p>
+            </li>
+          </a>
+        ))}
       </ul>
     </div>
   ) : (
     <NoResults query={currentRefinement} />
   );
 });
 
+// Pagination kinda sorta working
```

Comment:
should we remove the comments?

Reply:
Yes, sorry, let's remove this for now. I can look back at this PR later when we are redoing the Learn search page.